[2024-02-10T02:43:07.229+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_weather_data.extract manual__2024-02-10T02:42:54.586473+00:00 [queued]>
[2024-02-10T02:43:07.302+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_weather_data.extract manual__2024-02-10T02:42:54.586473+00:00 [queued]>
[2024-02-10T02:43:07.308+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2024-02-10T02:43:08.965+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): extract> on 2024-02-10 02:42:54.586473+00:00
[2024-02-10T02:43:08.985+0000] {standard_task_runner.py:60} INFO - Started process 113 to run task
[2024-02-10T02:43:09.037+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_weather_data', 'extract', 'manual__2024-02-10T02:42:54.586473+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp7vcjtif2']
[2024-02-10T02:43:09.068+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask extract
[2024-02-10T02:43:09.352+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_weather_data.extract manual__2024-02-10T02:42:54.586473+00:00 [running]> on host ff53158106ce
[2024-02-10T02:43:09.705+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Lucas Andrini' AIRFLOW_CTX_DAG_ID='etl_weather_data' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2024-02-10T02:42:54.586473+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-10T02:42:54.586473+00:00'
[2024-02-10T02:43:11.627+0000] {etl_dag.py:85} INFO - Error en la solicitud a la API para la ciudad (-34.61, -58.38).
[2024-02-10T02:43:13.119+0000] {etl_dag.py:85} INFO - Error en la solicitud a la API para la ciudad (-56.18, -34.9).
[2024-02-10T02:43:13.272+0000] {etl_dag.py:92} INFO - Contenido del DataFrame después de la extracción:
Empty DataFrame
Columns: []
Index: []
[2024-02-10T02:43:13.275+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-10T02:43:13.327+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_weather_data, task_id=extract, execution_date=20240210T024254, start_date=20240210T024307, end_date=20240210T024313
[2024-02-10T02:43:13.457+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-10T02:43:13.643+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
